packages/multichain/src/scope/assert.ts

```diff
@@ -0,0 +1,71 @@
+import { JsonRpcError } from '@metamask/rpc-errors';
+import type { Hex } from '@metamask/utils';
+
+import {
+  isSupportedMethod,
+  isSupportedNotification,
+  isSupportedScopeString,
+} from './supported';
+import type { InternalScopeObject, InternalScopesObject } from './types';
+
+export const assertScopeSupported = (
```

Comment:
Nit: More missing TSDocs for exported functions

Reply:
Added a bunch more TSDocs here: https://github.com/MetaMask/core/pull/4784/commits/6e345cf0c4a13329261720bf02ba6138538fa309